fidesctl/src/fideslang/models.py

```diff
@@ -487,6 +511,10 @@ class System(FidesModel):
     joint_controller: Optional[ContactDetails]
     third_country_transfers: Optional[List[str]]
     administrating_department: Optional[str] = "Not defined"
+    data_protection_impact_assessment: DataProtectionImpactAssessment = Field(
+        default=DataProtectionImpactAssessment(),
+        description=DataProtectionImpactAssessment.__doc__,
```

Comment:
Not sure if this is helpful in reducing duplication or just not a great idea 🤷🏽

Reply:
clever use of programmatic documentation :)